code/game/objects/items/stacks/sheets/glass.dm

```diff
@@ -358,7 +358,8 @@
 			var/obj/item/bodypart/O = H.get_bodypart(picked_def_zone)
 			if(!istype(O))
 				return
-			if(!H.shoes)
+			var/feetCover = (H.wear_suit && H.wear_suit.body_parts_covered & FEET) || (H.w_uniform && H.w_uniform.body_parts_covered & FEET)
+			if(!H.shoes && !feetCover)

```

Comment:
you could add ```H.shoes``` to ```feetCover``` to make this nicer.
also maybe make a proc, getFeetCoverage() or something, since technically this prevents a hat with feet coverage from working (yes it makes no sense, but it would *work* mechanically!)